workspace/apps/pidp/src/app/features/portal/portal.routes.ts

```diff
@@ -1,5 +1,6 @@
 export class PortalRoutes {
   public static BASE_PATH = 'portal';
+  public static readonly ACCESS_REQUEST_PAGE = 'accessrequest';
```

Comment:
I see you may have added this route declaration here to import into `portal-dashboard`, see comment below